Demos/Blazorise.Demo/Pages/Tests/SnackbarsPage.razor

```diff
@@ -157,15 +177,19 @@
     </SnackbarBody>
 </Snackbar>
 <SnackbarStack @ref="snackbarStack"
-               Location="SnackbarStackLocation.End"
-               DefaultInterval="3000"
+               Location="SnackbarStackLocation.TopEnd"
+               DefaultInterval="300000"
                DelayCloseOnClick />
-@code {
+
+ @code {
     Snackbar snackbar;
     Snackbar snackbarAction;
     Snackbar snackbarMultiline;
     Snackbar snackbarLeft;
     Snackbar snackbarRight;
+    Snackbar snackbarTop;
+    Snackbar snackbarTopEnd;
+    Snackbar snackbarTopStart;
```

Comment:
Could almost just have a configurable location parameter?

Reply:
Tried it. Makes it complicated without reason.